UI/src/model/LinearAssetsCollection.js

```diff
@@ -71,7 +71,7 @@
       });
     };
 
-    var fetch = function(boundingBox, assets) {
+    var fetch = function(boundingBox, assets, center) {
```

Comment:
The boundingBox parameter isn't used anymore on fetch function, can be removed from the parameters.